Formula/grafana.rb

```diff
@@ -13,7 +13,7 @@ class Grafana < Formula
   end
 
   depends_on "go" => :build
-  depends_on "node@10" => :build
```

Comment:
Nice

Reply:
Figured I would try if `node` 14 would work and it seems to be fine.